### ISSUE TITLE
epicgames: unknown error on main page

### ISSUE THREAD

**Issue by akorn** [ferdium/ferdium-app#2009]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that Ferdium Recipes adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22recipe+%3Abookmark_tabs%3A%22+label%3A%22bug+%3Abeetle%3A%22+) for a bug report that matches the one I want to file, without success.

### Ferdium Version

7.0.0

### Recipe Name

epicgames

### Recipe Version

1.3.1

### Last Known working version of the recipe

1.3.1

### Steps to reproduce

1. try to use the recipe
2. get redirected to https://www.fortnite.com/id/error?errorTitleCode=error.500.title&errorCode=error.invalidClient.wrongParameters

### Expected Behavior

The main page of the Epic Games Store should appear.

### Actual Behavior

An error message from Epic appears, saying "Sorry, the client that sent you here did not pass the parameters needed for you to login."

### Screenshots

![Screenshot_20241225_181541](https://github.com/user-attachments/assets/1adef0a9-b2e2-493d-8325-b5be15162004)

### Additional Information

I tried the following and neither helped:
 * clearing the cache
 * reloading the service
 * disabling and re-enabling the service
 * deleting and re-adding the service
 
I started seeing this behaviour about a week ago. Maybe Epic changed something and the recipe needs to be updated?

**Comment by laur89** [comment]
unsure whether it's related, but they recently updated EULA. log into your epic account via a browser and accept the new version if presented with one.

**Comment by akorn** [comment]
Nope, no new EULA (probably already accepted). Browser login works just fine; Ferdium still doesn't.

However, if I directly paste https://store.epicgames.com/en-US/ into the URL bar in Ferdium, it logs in just fine.

**Comment by Blkez** [comment]
I also get this error when I try to login via HTTPS://legendary.gl/epiclogin specifically when trying to connect my account with heroic games launcher on steam deck or lutris but also just by opening that URL on web browser anywhere. I do all login correctly, captcha correct and 2 step verification code, then get this error.

**Comment by supercaomi2077** [comment]
> I also get this error when I try to login via [HTTPS://legendary.gl/epiclogin](HTTPS://legendary.gl/epiclogin) specifically when trying to connect my account with heroic games launcher on steam deck or lutris but also just by opening that URL on web browser anywhere. I do all login correctly, captcha correct and 2 step verification code, then get this error.

Have you solved it? I'm having the same problem now and can't find a solution
